src/mscorlib/model.xml

```diff
@@ -6703,6 +6712,8 @@
       <Member Name="AddRef(System.IntPtr)" />
       <Member Name="AllocHGlobal(System.Int32)" />
       <Member Name="AllocCoTaskMem(System.Int32)" />
+      <Member Name="BindToMoniker(System.String)" />
+      <Member Name="ChangeWrapperHandleStrength(System.Object,System.Boolean)" />
```

Comment:
@weshaggard Do you know why are these two methods included? They are not in S.R.InteropServices and not in the master API list I've seen. These two APIs are not interesting.